.github/workflows/codeql.yml

```
@@ -40,16 +40,20 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v7.0.1
+<<<<<<< HEAD
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)